assets/css/extended/custom.css

```
@@ -5326,4 +5505,164 @@ pre:hover .copy-code {
+@media (max-width: 600px) {
+    .ai-tech-grid {
+        grid-template-columns: 1fr !important;
+    }
+}
+
+/* Ensure long words/URLs break in post content on mobile */
+@media (max-width: 768px) {
+    .post-content p,
+    .post-content li,
+    .post-content blockquote {
+        word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)